matplotlib2tikz/axes.py

```diff
@@ -389,7 +389,10 @@ def _get_ticks(data, xy, ticks, ticklabels):
         pgfplots_ticks.append(tick)
         # store the label anyway
         label = ticklabel.get_text()
-        pgfplots_ticklabels.append(label)
+        if ticklabel.get_visible():
+            pgfplots_ticklabels.append(label)
+        else:
```

Comment:
I don't understand the logic here. "If the label is not visible, it's required."?